sdv/multi_table/hma.py

```diff
@@ -629,6 +647,13 @@ def _get_likelihoods(self, table_rows, parent_rows, table_name, foreign_key):
             except (AttributeError, np.linalg.LinAlgError):
                 likelihoods[parent_id] = None
 
+        if f'__{table_name}__{foreign_key}' in self._null_child_synthesizers:
+            try:
+                likelihoods[np.nan] = synthesizer._get_likelihood(table_rows)
+
+            except (AttributeError, np.linalg.LinAlgError):
+                likelihoods[np.nan] = None
```

Comment:
what happens when we set the likelihoods to None?

Reply:
If at least one parent row has a valid likelihood, NaN likelihoods will be filled with the mean likelihood.  It shares logic with when a parent row has an error calculating likelihood